src/preact/base-element.js

```diff
@@ -671,6 +675,20 @@ function usesShadowDom(Ctor) {
   );
 }
 
+/**
+ * @param {null|string} attributeName
```

Comment:
I don't think this can ever be null? Was Closure complaining when it was just `string`?

Reply:
Yup, this was mainly so Closure doesn't complain. I don't think it can be null either.

Reply:
`MutationRecord`, I believe, defines `attributeName` as nullable.